StepDefinitions/FacebookSD/SignupSD.js

```diff
@@ -0,0 +1,93 @@
+const { Given, Then } = require("@wdio/cucumber-framework");
+const { expect } = require("chai");
+const moment = require("moment");
+const Homepage = require("../../Pages/Facebook/Homepage");
+const SignupPage = require("../../Pages/Facebook/SignupPage");
+
+Given(/^I am on facebook homepage$/, async function () {
+    await browser.url('/');
+    await browser.pause(5000);
+})
+
+Given(/^I click on Create New Account button$/, async function () {
+    const homepage = new Homepage();
+    const signuppage = new SignupPage();
+    await homepage.clickCreateNewAccountBtn();
+    await signuppage.waitForSignUpFormDisplay();
+    await browser.pause(2000);
+})
+
+Then(/^I verify current month is selected in month dropdown$/, async function () {
+    const signuppage = new SignupPage();
+    //Moment library code for current month
+    const currentMonth = moment().format('MMM');
+
+    //Facebook code for current month.
+    const selectedMonth = await signuppage.getCurrentSelectedMonth();
+
+expect(selectedMonth, 'Current Month Is NOT selected').to.equal(currentMonth);
+});
+
+Then(/^I verify current date is selected in day dropdown$/, async function () {
+   const signuppage = new SignupPage();
+    //Moment library code for current day
+    const currentDay = moment().format('D');
```

Comment:
Its better to have above code in a function in POM and call the func here.
Meaning no coding in SD files. All coding in POM.